industrial_taxonomy/pipeline/official/gdp/utils.py

```diff
@@ -0,0 +1,19 @@
+"""Clean and process GDP data"""
+
+import pandas as pd
+import re
+
+
+def process_gdp_table(table: pd.DataFrame):
+    """Cleans up the gdp table"""
+    _table = (
+        table.dropna(
+            axis=0, subset=["LA code"]  # We are dropping bottom rows without a LA code
+        )
+        .rename(columns={"2019\n[note 3]": "2019"})
+        .rename(columns={x: str(x) for x in range(1998, 2020)})
```

Comment:
Perhaps something to address later: Will this be a common date range for datasets and later analysis?